src/localization/locales/en.json

```diff
@@ -100,7 +100,7 @@
             "interfaces": "Interface",
             "events": "Telemetry",
             "eventsPerInterface": "Telemetry",
-            "methods": "Device Methods",
+            "methods": "Direct Method",
```

Comment:
Wonder if this should be changed to "Direct methods" to follow the capitalization of the other menu items.

Reply:
Please check elsewhere too

Reply:
Good catch! I was only paying attention to the portal...